src/components/LessonPlanList/LessonPlanRow/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+
+import { withStore } from '../../../Store';
+
+class LessonPlanRow extends Component
+{
+  onClick = event =>
+  {
+    const lessonPlan = this.props.lessonPlan;
+    const store = this.props.store;
+    store.updateCurrentLessonPlanID(lessonPlan.id);
+    delete lessonPlan.id;
```

Comment:
Mutating props like this can mess things up. Make a copy of the data before updating

Reply:
Fixed